stories/molecules/Table.mdx

```diff
@@ -37,15 +36,16 @@ const People = () => <Table columns={columns} data={data} />;
 
 The `columns` prop config can take a number of more advanced configurations as seen above. For more advanced configuration you can supply additional properties to the `columns` array:
 
-| property          | type       | description                                                                                                                                             |
-| ----------------- | ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| key               | `String`   | Key representing the corresponding data property.                                                                                                       |
-| children          | `Node`     | Custom markup to render in the column heading [See example](#custom-column-markup)                                                                      |
-| disableSorting    | `Boolean`  | Disables ability to sort column data                                                                                                                    |
-| dataCellDecorator | `Function` | Customize cell markup with a decorator, a function that is called with the cell data, a React key, and the row data. [See example](#custom-cell-markup) |
-| headingProps      | `Object`   | Props object to be passed to the `<TableHeading>` component. [See example](#custom-heading-props)                                                       |
-| sortCompareAsc    | `Function` | Custom ascending sort compare function. [See example](#custom-sorting)                                                                                  |
-| sortCompareDesc   | `Function` | Custom descending sort compare function. [See example](#custom-sorting)                                                                                 |
```

Comment:
I adjusted the "see example" links here as they weren't working, just an FYI that markdown tables don't render in storybook